src/app/theme/index.tsx

```diff
@@ -13,17 +13,27 @@ import texGyeAdventorRegularWoff2 from 'assets/font/tex-gyre-adventor-regular/te
 // Update your breakpoints if you want
 export const sizes = {
   upExtraSmall: 600,
+  aBitSmall: 650,
   upSmall: 800,
   upMedium: 1000,
   upLarge: 1200,
 };
 
+const maxWidth = ['aBitSmall'];
+
 export const media = (Object.keys(sizes) as Array<keyof typeof sizes>).reduce((acc, label) => {
-  acc[label] = (first: any, ...interpolations: any[]) => css`
-    @media (min-width: ${sizes[label]}px) {
-      ${css(first, ...interpolations)}
-    }
-  `;
+  acc[label] = (first: any, ...interpolations: any[]) =>
+    maxWidth.includes(label)
+      ? css`
+          @media (max-width: ${sizes[label]}px) {
+            ${css(first, ...interpolations)}
+          }
+        `
+      : css`
+          @media (min-width: ${sizes[label]}px) {
+            ${css(first, ...interpolations)}
+          }
+        `;
```

Comment:
@WeiLi512 I've used this way => Because we only update width for risk ratio tooltips on SP, It seems we should not be used the mobile-first approach. Please check it again. Thank you!